conftest.py

```diff
@@ -91,9 +91,27 @@ def pytest_fixture_setup(fixturedef: FixtureDef[Any], request: FixtureRequest) -
 
 
 def pytest_runtest_setup(item: Item) -> None:
+    """
+    Performs the following actions:
+    1. Adds skip fixture for kserve if serverless or authorino operators are not installed.
+    2. Adds skip fixture for serverless if authorino/serverless/service mesh are not deployed.
+    """
+
     BASIC_LOGGER.info(f"\n{separator(symbol_='-', val=item.name)}")
     BASIC_LOGGER.info(f"{separator(symbol_='-', val='SETUP')}")
 
+    if KServeDeploymentType.SERVERLESS.lower() in item.keywords:
+        item.fixturenames.insert(0, "skip_if_no_redhat_authorino_operator")
```

Comment:
nitpick, but shouldn't it be called `skip_if_no_deployed...` like the other two?